keyboards/lazydesigners/the30/readme.md

```diff
@@ -0,0 +1,17 @@
+# THE30
+
+![the30](https://i.loli.net/2019/11/06/KTn3dNjreigovRz.jpg)
+
+A 30% custom keyboard designed and produced by [LazyDesigners](http://lazydesigners.cn).
+
+Keyboard Maintainer: [Erovia](https://github.com/Erovia)  
+Hardware Supported: THE30  
+Hardware Availability: Check for GBs on [Geekhack](https://geekhack.org) and on [LazyDesigner's homepage.](http://lazydesigners.cn)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make lazydesigners/the30:default:dfu
```

Comment:
```suggestion
    make lazydesigners/the30:default
```

Reply:
Why remove the flashing info?
(changing it to `:flash` would be better, though)

Reply:
This is just supposed to be the make example - good point though, there should be an equivalent flash command example in the template as well.

Reply:
Two entries then?
One for make and one for flashing?

Reply:
Yep, see the NIU Mini's readme for a good format: https://github.com/qmk/qmk_firmware/tree/master/keyboards/niu_mini

Reply:
I actually like something like:

```Markdown
    make lazydesigners/the30:default        # Compile
    make lazydesigners/the30:default:flash  # Compile and flash
```

I'm not attached to that format, though.